deeplake/util/transform.py

```diff
@@ -537,6 +541,12 @@ def process_transform_result(result: List[Dict]):
     return final
 
 
+def _get_avg_chunk_size(num_samples, max_shape, num_chunks, dtype):
```

Comment:
can we make this a property of chunk engine? can be useful elsewhere. We have access to all this information there too.

Reply:
done